REVIEWING.md

```diff
@@ -119,7 +120,10 @@ This makes any change that can impact the Cluster API contract critical and usua
 
 ### Logging
 
-- For CAPI controllers see [Kubernetes logging conventions](https://git.k8s.io/community/contributors/devel/sig-instrumentation/logging.md).
+While developing controllers in Cluster API it is required add logging so the system can be observed
+and all the required info is available when troubleshooting issues.
+
+- For CAPI controllers see TODO.
```

Comment:
Can we just force merge with the correct link? This is pretty bad UX for adding pages to the book!

Reply:
REVIEWING.md is an edge case, because this page is defined outside of the book but also imported in the book.
So, in order to make this page to work from outside the book you need to use the generated url of the target page https://cluster-api.sigs.k8s.io/developer/logging.html, that will exists only when the book is merged.